File: ALOD.CS/Public/Ping.ashx.cs
using System;
using System.Web;

namespace ALOD
{
    /// <summary>
    /// Summary description for Ping
    /// </summary>
    public class Ping : IHttpHandler
    {
        public bool IsReusable
        {
            get
            {
                return false;
            }
        }

        public void ProcessRequest(HttpContext context)
        {
            context.Response.ContentType = "text/plain";
            context.Response.Write(DateTime.Now.Ticks);
        }
    }
}

File: ALOD/Public/Ping.ashx.vb
Public Class Ping
    Implements System.Web.IHttpHandler

    ReadOnly Property IsReusable() As Boolean Implements IHttpHandler.IsReusable
        Get
            Return False
        End Get
    End Property

    Sub ProcessRequest(ByVal context As HttpContext) Implements IHttpHandler.ProcessRequest

        context.Response.ContentType = "text/plain"
        context.Response.Write(Now.Ticks)

    End Sub

End Class